### PR TITLE
pdftocairo: add page

### PR DESCRIPTION
**Pull request by mt-empty**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pdftocairo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`pdftocairo {{path/to/file}}.pdf {{output.png}} -png -r 200`
+
+- Convert to grayscale TIFF setting paper size to A3:
+
+`pdftocairo {{path/to/file}}.pdf -tiff -gray -paper A3`
+
+- Convert to PNG cropping x and y pixels from the top left corner:
+
+`pdftocairo {{path/to/file}}.pdf -png -x {{number}} -y {{number}}`
```
Comment (zdroid) [inline pages/common/pdftocairo.md]: -x number and -y number should have different names, for example `x_pixels` and `y_pixels`.

File: pages/common/pdftocairo.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`pdftocairo {{path/to/file}}.pdf -jpeg`
+
+- Convert to PDF expanding the output to fill the paper:
+
+`pdftocairo {{path/to/file}}.pdf {{output.pdf}} -pdf -expand`
+
+- Convert to SVG specifying the first/last page to convert:
+
+`pdftocairo {{path/to/file}}.pdf {{output.svg}} -svg -f {{number}} -l {{number}}`
```
Comment (zdroid) [inline pages/common/pdftocairo.md]: The two `number`s should have different names, e.g. `first_page` and `last_page`.

File: pages/common/pdftocairo.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# pdftocairo
+
+> Converts PDF files to PNG/JPEG/TIFF/PDF/PS/EPS/SVG using cairo.
+> More information: <https://poppler.freedesktop.org>.
+
+- Convert a PDF file to JPEG:
+
+`pdftocairo {{path/to/file}}.pdf -jpeg`
```
Comment (zdroid) [inline pages/common/pdftocairo.md]: I think `{{path/to/file}}.pdf` should be replaced with `{{path/to/file.pdf}}` so the syntax can be properly highlighted.

Comment (mt-empty) [reply pages/common/pdftocairo.md]: Thanks for your feedback, will be making the changes soon.
meanwhile when do I use this {{path/to/file}}.pdf then?

**Comment by mt-empty** [comment]
@zdroid 
Thanks for your feedback, 
When do I use this {{path/to/file}}.pdf?

File: pages/common/pdftocairo.md (hunk 3 above)
Comment (zdroid) [reply pages/common/pdftocairo.md]: I think it's best to be avoided. As far as I've seen so far, arguments are either completely inside the brackets or completely outside them.

**Pull request merged**